.gitattributes

```diff
@@ -0,0 +1,2 @@
+brave-*.txt linguist-language=AdBlock linguist-detectable
+coin-*.txt linguist-language=AdBlock linguist-detectable
```

Comment:
@ryanbr @ShivanKaul can we move `debounce.json` out of `brave-lists`, and then move `brave-unbreak.txt` and `coin-miners.txt` into `brave-lists`? That should make the organization here a bit simpler.

Reply:
In favour of moving `brave-unbreak.txt` and `coin-miners.txt` into `brave-lists`, but not sure where else `debounce.json` could live because it is technically a `brave-list`. Maybe we could have a sub-folder here with all the true adblock lists?

Reply:
That sounds reasonable. I'll note this down in a new issue and we can handle it in a followup.